server/common.js

```diff
@@ -177,6 +179,49 @@ var getDate = function () {
 }
 exports.getDate = getDate;
 
+/**
+* return the current date
+*
+* @return {date}
+*/
+var getDateObject = function(){
+    return new Date();
+}
+exports.getDateObject = getDateObject;
+
+/**
+* return the time in day hours minutes and seconds format
+*
+* @return {string}
+*/
+var getTime = function(time){
+    var date = new Date(time);
+    var str = '';
+    var days = date.getUTCDate()-1;
+    var hours = date.getUTCHours();
+    var minutes = date.getUTCMinutes();
+    var seconds = date.getUTCSeconds();
+    if (days){
+        str += date.getUTCDate()-1 + " days, ";
+    }
+    if (hours){
+        str += date.getUTCHours() + " hours, ";
+    }
+    if (minutes){
+        str += date.getUTCMinutes() + " minutes, ";
+    }
+    if (seconds){
+        str += date.getUTCSeconds() + " seconds";
+    }
+    return str;
+}
+exports.getTime = getTime;
+
+// return the current date with format
+var getDateByFormat = function(format) {
```

Comment:
You added 3 new date functions, not sure if you need them all. There were others implemented that you can look at to see if you can reuse?

Reply:
I added 2 not 3, the other one was already there I believe. Looks like I copied and pasted the same function... will delete